src/content/docs/zh-cn/recipes.mdx

```diff
@@ -0,0 +1,33 @@
+---
+title: 更多方案
+description: 为Astro添加特性的示例。
+
+---
+
+import RecipesNav from '~/components/RecipesNav.astro';
+
+为你的Astro项目中添加特性的示例。
+
+
+## 官方方案
+
+<RecipesNav />
+
+## 社区资源
+
+晒出你的方案! 更多详情请查看 [recipes contributing guide](https://github.com/withastro/docs/blob/main/RECIPES.md)。
```

Comment:
The exclamation mark should be represented by Chinese punctuation, and the link description should also be translated:
```suggestion
晒出你的方案！更多详情请查看[示例贡献指南](https://github.com/withastro/docs/blob/main/RECIPES.md)。
```